### DISCUSSION TITLE
import Polygon from 'ol/geom/Polygon';

### DISCUSSION THREAD

**Discussion by tjtyler** [tethysplatform/tethys#773] [Q&A]
When i try to
import Polygon from 'ol/geom/Polygon';
at the top of my main.js file, I get this error in my console: Uncaught SyntaxError: Cannot use import statement outside a module.
I've noticed too that in this example (https://openlayers.org/en/latest/examples/center.html), that they have a package.json file but I don't see that in my tethysapp directory. do I need to make a package.json file or do I need to do something else in oder to import Polygon?

**Comment by swainn** [comment]
JavaScript modules are a relatively new syntax for JavaScript. When using JavaScript without modules, all the variables in one script are available globally in any script that is loaded after it. When you declare a script as a module, then it requires that you import things you want to use from other scripts and export things you want to be importable to other scripts. Unfortunately, Tethys doesn't support module syntax yet, so you'll need to convert it to the global style.

For your example, you'll need to remove the import statement and replace "Polygon" with "ol.geom.Polygon" everywhere it is used in the code.
